src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java

```diff
@@ -281,6 +304,8 @@ private void checkTreesToFell()
          */
         public final Map<ItemStorage, Boolean> treesToFell = new LinkedHashMap<>();
 
+        public boolean shouldReplant = true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make shouldReplant a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)